account_financial_report/report/trial_balance.py

```diff
@@ -300,6 +300,29 @@ def _compute_partner_amount(
                         ] += round(tb["amount_currency"], 2)
         return total_amount, partners_data
 
+    def _remove_accounts_at_cero(self, total_amount, show_partner_details):
+        def is_removable(d):
+            return (
+                d["initial_balance"] == 0
```

Comment:
This should be used with `float_is_zero` (and if not possible to get decimal precision, put 8 digits).